test/junit/scala/util/ChainingOpsTest.scala

```diff
@@ -23,12 +23,10 @@ class ChainingOpsTest extends RunTesting {
 
   @Test
   def testAnyPipe: Unit = {
-    val times6 = (_: Int) * 6
-    val result = (1 - 2 - 3)
-      .pipe(times6)
-      .pipe(scala.math.abs)
+    val plus3 = (_: Int) + 3
+    val result = (1 - 2) |> plus3 |> scala.math.abs
 
-    assertEquals(24, result)
+    assertEquals(2, result)
```

Comment:
Why was the math changed?

Reply:
I put this in the PR description.
Because `|> times6 |> scala.math.abs` yields the same as `|> scala.math.abs |> times6`, thus introducing ambiguity.